libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -629,29 +635,29 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
             tooltipPlacement={'bottom-start'}
             optionalClassName="mr-3"
           />
-          
+
           <RemixUiCheckbox
             id="solhintstaticanalysis"
             inputType="checkbox"
-            title="Run solhint static analysis."
+            title="Run SolHint static analysis."
             onClick={handleLinterEnabled}
-            checked={solhintEnabled}
+            checked={solhintEnabled }
             label="Linter"
             onChange={() => {}}
             tooltipPlacement={'top-start'}
             optionalClassName="mr-3"
           />
-          
+
           <RemixUiCheckbox
             id="enableSlither"
             inputType="checkbox"
             onClick={handleSlitherEnabled}
-            checked={showSlither}
-            disabled={true}
+            checked={showSlither && slitherEnabled}
+            disabled={slitherEnabled}
             label="Slither"
             onChange={() => {}}
             optionalClassName="mr-3"
-            title="To run Slither analysis, Remix IDE must be connected to your local filesystem with remixd."
+            title={slitherEnabled ? "Run Slither static analysis." : "To have slither active, you need to have remixd connected to your local filesystem"}
```

Comment:
Update to:
To run Slither analysis, Remix IDE must be connected to your local filesystem with Remixd.

Reply:
done